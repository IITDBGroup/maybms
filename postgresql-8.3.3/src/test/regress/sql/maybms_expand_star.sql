--test for expanding star with functions and repair-key

create table s (a int);

create table r as repair key a in s;

select * from r;

select * from (select tconf() from r) s;

repair key (tconf) in (select * from (select tconf() from r) s);

repair key (a) in (select * from (select tconf() from r) s);

repair key (a) in (select * from (select a, tconf() from r) s);

drop table r;
drop table s;
