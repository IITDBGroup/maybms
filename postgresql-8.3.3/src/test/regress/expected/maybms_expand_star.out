--test for expanding star with functions and repair-key
create table s (a int);
create table r as repair key a in s;
select * from r;
 a | _v0 | _d0 | _p0 
---+-----+-----+-----
(0 rows)

select * from (select tconf() from r) s;
 tconf 
-------
(0 rows)

repair key (tconf) in (select * from (select tconf() from r) s);
 tconf | _v0 | _d0 | _p0 
-------+-----+-----+-----
(0 rows)

repair key (a) in (select * from (select tconf() from r) s);
ERROR:  column "a" does not exist
LINE 1: repair key (a) in (select * from (select tconf() from r) s);
                    ^
repair key (a) in (select * from (select a, tconf() from r) s);
 a | tconf | _v0 | _d0 | _p0 
---+-------+-----+-----+-----
(0 rows)

drop table r;
drop table s;
